src/components/Settings/SettingsSortGroups.vue

```diff
@@ -0,0 +1,77 @@
+<!--
+  - @copyright Copyright (c) 2020 Uriel Fontan <urielfontan2002@gmail.com>
+  -
+  - @author Uriel Fontan <urielfontan2002@gmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<div class="sort-groups">
+		<multiselect
```

Comment:
```suggestion
		<Multiselect
```

Reply:
thats actually interesting, SettingsSortContacts actually has the same multiselect, im not a vue genius so im interested in the reasoning behind this

Reply:
I guess this is because we register the Multiselect globally instead of on a per file basis (which we should do to be fair)
Vue recommend using PascalCase for components in template to differentiate from regular html elements :)

This is just semantics, nothing to worry about